### DISCUSSION TITLE
articles/the-greatest-git-commit

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#123] [Comments]
https://hutusi.com/articles/the-greatest-git-commit 

 TL;DR 本文较长，如果对 Git 内部实现不感兴趣可以快速跳过中间两个章节。

**Comment by petite-dd** [comment]
被写得太专业劝退了，哈哈，这种东西适合早上看。
看到博友能写得这么细致和专业，状态非常好呀，祝一切顺利~

**Reply by webersongao** [reply]
哈哈哈，适合早上看，是什么梗儿？

**Comment by hutusi** [comment]
> @petite-dd
> 被写得太专业劝退了，哈哈，这种东西适合早上看。
> 看到博友能写得这么细致和专业，状态非常好呀，祝一切顺利~

哈哈，谢谢。最近想写点长文~~~

**Comment by suguoyao** [comment]
学习啦👍

**Comment by neverlibis** [comment]
Page tail tip Error: Network Error

**Comment by hutusi** [comment]
> @neverlibis
> Page tail tip Error: Network Error
> 

是这个评论系统报的错吗？

**Comment by Junyi-99** [comment]
太棒了，文章清晰易懂！希望作者继续加油！准备去看看git的第一次commit是有多么精妙了！文章最后提出的MVP概念，在创业中也经常遇到。

总之，很赞！

**Comment by hutusi** [comment]
> @Military-Doctor
> 太棒了，文章清晰易懂！希望作者继续加油！准备去看看git的第一次commit是有多么精妙了！文章最后提出的MVP概念，在创业中也经常遇到。
> 
> 总之，很赞！

谢谢 :)

**Comment by robertu7** [comment]
寫的很棒，感謝。

**Comment by hutusi** [comment]
@robertu7 谢谢，你的网站和作品设计的都很好看，简洁。

**Comment by GaryLiu1996** [comment]
向前辈学习

**Comment by hutusi** [comment]
@GaryLiu1996 ”前辈“这个称呼听起来像是”前浪“😂

**Comment by hutusi** [comment]
@suguoyao 看了下你的网站首页，也很酷呀👍

**Comment by tizee** [comment]
好文，赞了。
